app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -279,7 +279,7 @@ def self.raw_alerts_connect(hostname, port, username, password)
         :username => username,
         :password => password
       }
-      ::Hawkular::Alerts::AlertsClient.new(url, credentials)
+      ::Hawkular::Alerts::Client.new(url, credentials)
```

Comment:
Can you have a problem in middleware since you are not setting the tenant explicitly?

I'm referring to:
https://github.com/hawkular/hawkular-client-ruby/blob/master/api_breaking_changes.rdoc#200